compiler/rustc_middle/src/mir/interpret/error.rs

```diff
@@ -493,7 +493,7 @@ impl dyn MachineStopType {
 }
 
 #[cfg(all(target_arch = "x86_64", target_pointer_width = "64"))]
-static_assert_size!(InterpError<'_>, 64);
+static_assert_size!(InterpError<'_>, 88);
```

Comment:
Do we even care about the size of this type? We care about `InterpErrorInfo` but that is a `Box` so it is independent of  the size of `InterpError`.

So I think I would favor removing this assertion, it might just lead someone to unnecessarily introduce another unnecessary `Box` indirection.